.devcontainer/devcontainer.json

```diff
@@ -1,20 +1,20 @@
 {
     "portsAttributes": {
-        "8080": {
+        "3000": {
             "label": "Application",
             "onAutoForward": "openPreview"
         }
     },
     "customizations": {
         "codespaces": {
             "openFiles": [
-                "build/app.js"
```

Comment:
`build` is the built artifacts, not the files they should edit, so we're ignoring them below and we should send them to ... I guess game coordinator? Or we could open both the index.html and this?